### DISCUSSION TITLE
2个月没说话，群咋不对我开放了？

### DISCUSSION THREAD

**Discussion by ni-feng-le** [0-8-4/miui-auto-tasks#262] [综合讨论｜General]
没说过话，就没了

**Comment by nimafengle** [comment]
干了啥你他妈的心里没点b数吗

**Reply by ni-feng-le** [reply]
你有毛病是不是，我干什么了，有什么数说清楚，看不懂大标题？

**Reply by ni-feng-le** [reply]
这哪家的那个玩意，放出来乱咬人，就你这没没素质的还还隔着公共处乱说话，你是不是小学没毕业

**Comment by nimafengle** [comment]
你妈的连着几天让群友帮你干这干那，折磨完群友还拿脚本去卖钱 是你吧

**Reply by ni-feng-le** [reply]
你他妈的傻逼吧，我自己都没整会我赚什么钱，你是不是脑残，我连服务器都没买赚你妈呢，艹了，你个大傻逼

**Comment by nimafengle** [comment]
![IMG_20240204_114922_026.jpg](https://github.com/0-8-4/miui-auto-tasks/assets/153733182/3f1caf44-94da-4352-a27e-f1d18e411d05)

这个是你吧

**Reply by ni-feng-le** [reply]
这是哪里呢，我只在酷安发过一次删了，你踏马随便捡个图就说是我，人家这个明明是成功了的，我都没成功过，硬逼着给我？你个大傻逼，赶紧给我道歉

**Comment by ni-feng-le** [comment]
没见过你这种狗，不进就不进了，真踏马给你脸了，第一次问问题稍微问多了点就说话很难听那没问题是我请教你们的我踏马忍了，完事还没教完又给群主说把我踢出去，然后又给群主解释的跟个狗一样，群主不计较又进来，后面一直就没有说过话，没想到临了又碰见你个大傻逼大狗隔着阴我，我操你妈去你大爷的吧，群我不进了，进你妈，靠你奶奶个腿

**Reply by BA7JCM** [reply]
> 问题稍微问多了点就说话很难听

你也好意思说，你咋不看看你都干了些啥，什么资料都不去查，脑子也不带，就带了张嘴在那里什么都问，问个问题跟别人欠你的一样，合着你有问题我们都必须得给你解答是吧，你谁啊你，真当自己算根葱了还

**Reply by KiwiR1der** [reply]
讨口子是这样的☺️

**Reply by BA7JCM** [reply]
伸手能伸的这么大言不愧也是要点本事的，脸皮没厚到一定程度还真干不出来这种逼事

**Comment by nimafengle** [comment]
看来没骂错，就你这吊样早就想骂了，什么玩意

**Comment by ni-feng-le** [comment]
狗先咬的我

**Comment by ni-feng-le** [comment]
说破天我都站理，就你这个狗乱咬人

**Comment by nimafengle** [comment]
我知道你很急，但是你写别急
我去找找你的经典语录
让大伙鉴赏一下

**Reply by ni-feng-le** [reply]
滚你麻类隔壁的吧，他妈的你个狗再给我说话你全家死完，我愿意搭理你？

**Comment by nimafengle** [comment]
![Screenshot_2023-12-13-20-07-15-907-edit_org.telegram.messenger.jpg](https://github.com/0-8-4/miui-auto-tasks/assets/153733182/8c98a9ab-9055-4304-adbc-47eee6da53b7)

![Screenshot_2023-12-13-20-08-08-182-edit_org.telegram.messenger.jpg](https://github.com/0-8-4/miui-auto-tasks/assets/153733182/faca8221-d640-4908-bd39-272628d20dbd)

![Screenshot_2023-12-13-20-07-41-642-edit_org.telegram.messenger.jpg](https://github.com/0-8-4/miui-auto-tasks/assets/153733182/0cbb2d9f-43f2-454f-ba2f-5c5050f541ff)

![Screenshot_2023-12-13-20-08-27-356-edit_org.telegram.messenger.jpg](https://github.com/0-8-4/miui-auto-tasks/assets/153733182/6708bc0f-af8e-4df2-8ef8-1445f13cdf82)

![Screenshot_2023-12-13-20-10-16-193-edit_org.telegram.messenger.jpg](https://github.com/0-8-4/miui-auto-tasks/assets/153733182/96fedf34-1950-47bb-bef2-088b44251bbd)

![Screenshot_2023-12-13-20-10-27-773-edit_org.telegram.messenger.jpg](https://github.com/0-8-4/miui-auto-tasks/assets/153733182/662b0d3d-53fe-49e3-826c-2f23a788e0f0)

![Screenshot_2023-12-13-20-11-10-765-edit_org.telegram.messenger.jpg](https://github.com/0-8-4/miui-auto-tasks/assets/153733182/8f8f7fd1-bef6-4e6c-9c21-47078a4a364c)

![Screenshot_2023-12-13-20-11-57-716-edit_org.telegram.messenger.jpg](https://github.com/0-8-4/miui-auto-tasks/assets/153733182/abdfc265-fe2c-44e7-8332-60a08b5ff99e)

![Screenshot_2023-12-13-20-13-08-588-edit_org.telegram.messenger.jpg](https://github.com/0-8-4/miui-auto-tasks/assets/153733182/e33485bc-e336-4e31-8237-34c66a8f71ef)

![Screenshot_2023-12-13-20-15-41-921-edit_org.telegram.messenger.jpg](https://github.com/0-8-4/miui-auto-tasks/assets/153733182/aa4421f1-c3be-453b-ab5e-f474dfaa5ec8)

![Screenshot_2023-12-13-20-20-10-734-edit_org.telegram.messenger.jpg](https://github.com/0-8-4/miui-auto-tasks/assets/153733182/33f203ca-a157-46ba-a85b-22b300321f0d)

![Screenshot_2023-12-13-20-19-54-634-edit_org.telegram.messenger.jpg](https://github.com/0-8-4/miui-auto-tasks/assets/153733182/568242aa-3712-4947-86f6-9328dcfa8e38)

![Screenshot_2023-12-13-20-19-39-006-edit_org.telegram.messenger.jpg](https://github.com/0-8-4/miui-auto-tasks/assets/153733182/b4c3c465-69d9-4e5b-9e5f-50bddf9b710a)

你继续

**Reply by ni-feng-le** [reply]
呵呵，煞笔 he——tui！

**Reply by 1sony** [reply]
这是tg吗？怎么进

**Comment by nimafengle** [comment]
一小时过去了，怎么还没等到你大段的狡辩啊，你这骂完就结束了？

**Comment by koolcandy** [comment]
人呢,怎么不回了,瓜还没吃够啊

**Comment by BA7JCM** [comment]
咋不吱声了，接着闹腾啊 @ni-feng-le 别停下来啊，你不是觉得自己一点错都没有吗

**Comment by nimafengle** [comment]
![Screenshot_2023-12-13-20-11-57-716-edit_org.telegram.messenger.jpg](https://github.com/0-8-4/miui-auto-tasks/assets/153733182/8581ea5b-5752-421f-aa7b-fbea1a0dc459)

![Screenshot_2024-02-04-14-54-22-960_com.magicalstory.AppStore.jpg](https://github.com/0-8-4/miui-auto-tasks/assets/153733182/08931e93-3d9e-412a-8a5b-5d35fcb69a94)

![Screenshot_2024-02-04-14-54-41-701_com.tencent.mobileqq.jpg](https://github.com/0-8-4/miui-auto-tasks/assets/153733182/f2394764-c28b-48a7-a445-87167b8c86ea)

![Screenshot_2024-02-04-14-54-51-093_com.tencent.mobileqq.jpg](https://github.com/0-8-4/miui-auto-tasks/assets/153733182/889172df-2e96-4193-8361-227e1377d0e8)

还需要我继续帮你回忆吗，亲爱的贾先生？

**Reply by ni-feng-le** [reply]
靠你妈类隔壁，你个傻逼那是别人的群，部分青红皂白就他妈乱冤枉人你妈类隔壁，你去群里哦看了吗，我在里面发的全是游戏辅助，煞笔玩意，你妈死了

**Reply by BA7JCM** [reply]
> 你个傻逼那是别人的群

“别人的群”，感情你用别人的QQ号作为你小米账号的密码是吧，之前你在群里提供的密码里的QQ号就是你在奇妙应用上面的那个卖代挂的帖子里留下的QQ群的群主，而是多个平台多个部位你都用相同的网名或者说ID，设备也相同，部署的平台也相似，这么多的巧合，如果不是你，那它们真的就只是“巧合”吗？快三十岁的人了，成熟点，别一天到晚像个小学生一样，老大不小了，95年十一月上旬到现在，你都28岁了，8岁小孩都没你这么不要脸，拿别人免费的东西去卖钱，被抓现行还死活不认，要点脸吧！

**Comment by ni-feng-le** [comment]
你踏马的说那么多干叼毛，你说那么多你以为我都愿意进你那个笔群了？傻逼玩意，你自己把群留好，赶明群生群给你传宗接代

**Comment by ni-feng-le** [comment]
He-tui

**Comment by nimafengle** [comment]
我看你很急啊，脑子急坏了没办法思考了是吧
能不能找个好一点的借口啊，你这群主说自己的群不是你的群？你从你妈肚子里出来要不要说你妈不是你妈啊？

**Reply by ni-feng-le** [reply]
人家不露面懂不懂，给傻逼说话真费劲，he-tui恶心

**Comment by ni-feng-le** [comment]
呦呦，这哪又出来个舔狗，白嫖让你们相遇，在群里搞搞舔着不好吗，查你爹信息是想吃牢饭了？我就说群里面一大堆高中生初中生，赶紧把群藏好 把群供起来当你祖宗养着先传个十八代再放出来，供奉好了别出来丢人现眼，煞笔

**Comment by nimafengle** [comment]
怎么看你越来越急了，你先别急啊
你这前面一会儿说不知道奇妙应用是个啥平台，一会儿又说是你帮别人代发的
不狡辩下？

**Comment by ni-feng-le** [comment]
人家大哥根本不用你们的破玩意好吧人家自己是实力别把自己太当回事，你好像条狗啊，he-tui守着你的群传宗接代去吧

**Reply by BA7JCM** [reply]
动动你仅剩的一点点猪脑子想想，群里那么多人为何偏偏就你被踹了出来而且还被加入了黑名单，同时被几乎所有当时在场的人觉得烦，我可以明确告诉你，截至目前，你是第一个，也是唯一的一个，被群里人一致要求群主踢掉并拉黑的人，没有第二个人有你这个待遇，快三十岁的人了，成熟点

**Comment by ni-feng-le** [comment]
煞笔

**Comment by nimafengle** [comment]
太对了
打广告把开源地址都打出来了
看来我前面说的是对的，你妈果然不是你妈

**Comment by xuan06zyx** [comment]
什么古战场，开源社区不得不品的一环
